CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ This release contains a number of fixes and improvements:
 
```

Comment:
This is missing the container list in pod list feature, was only in a PR that was not marked with the 3.1 milestone. Have added now. Shall I manually add it this time round?

Reply:
I'll update the PR with the tool